src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -795,13 +924,13 @@ std::pair<Pane::PanePoint, Pane::PanePoint> Pane::_GetOffsetsForPane(const PaneP
 
     if (_splitState == SplitState::Horizontal)
     {
-        secondOffset.y += (1 - _desiredSplitPosition) * parentOffset.scaleY;
+        secondOffset.y += _desiredSplitPosition * parentOffset.scaleY;

```

Comment:
note to self: did `_desiredSplitPosition` get inverted

Reply:
Nope, I had a bug originally :) Trying to put the logic into english, it is supposed to be "advance position to the split position", and before this change it was doing "advance position by the remaining size"

Pointed this out in a comment above, but it was at the very top so probably got lost by all of the other comments.
https://github.com/microsoft/terminal/pull/11153#discussion_r702928256